src/IKVM.Runtime.JNI/JNIEnv.cs

```diff
@@ -349,8 +349,7 @@ internal static string StringFromOEM(byte* psz)
             {
                 if (psz[i] == 0)
                 {
-                    int oem = System.Globalization.CultureInfo.CurrentCulture.TextInfo.OEMCodePage;
-                    return new String((sbyte*)psz, 0, i, Encoding.GetEncoding(oem));
+                    return new String((sbyte*)psz, 0, i, Encoding.Default);
```

Comment:
I changed this here to Encoding.Default, because JNI[^1] uses this `StringFromOEM` for creating the JVM Args.
[^1]: https://github.com/ikvm-revived/ikvm/blob/0f894be58dcc287ff996a87cfa64ade5917f0780/src/IKVM.Runtime.JNI/JNI.cs#L219

Didn't want to create any issues with Sun UTF-8 encoding here, as that might introduce other issues.

Reply:
Encoding.Default is definitely wrong too. The spec says modified UTF-8. There's a function for that.